.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+.nyc_output
```

Comment:
@Qix- I think the intention is to merge #4 first, at which point commit 4181246 will no longer be a part of this PR. If that's acceptable, then we should essentially ignore the changes from that commit here.